src/wp-includes/fonts/class-wp-font-face.php

```diff
@@ -389,6 +379,58 @@ private function build_font_face_css( array $font_face ) {
 		return $css;
 	}
 
+	/**
+	 * Normalizes a font-face name for use in CSS.
+	 *
+	 * Add quotes to the font-face name and escape problematic characters.
+	 *
+	 * @see https://www.w3.org/TR/css-fonts-4/#font-family-desc
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param string $font_family The font-face name to normalize.
+	 * @return string The normalized font-face name.
+	 */
+	protected function normalize_css_font_family( string $font_family ): string {
+		$font_family = trim( $font_family, " \t\r\f\n" );
+
+		if (
+			strlen( $font_family ) > 1 &&
+			( '"' === $font_family[0] && '"' === $font_family[ strlen( $font_family ) - 1 ] ) ||
+			( "'" === $font_family[0] && "'" === $font_family[ strlen( $font_family ) - 1 ] )
+		) {
+			_doing_it_wrong(
+				__METHOD__,
+				__( 'Font family should not be wrapped in quotes; they will be added automatically.' ),
+				'6.9.0'
+			);
+			return $font_family;
+		}
+
+		return '"' . strtr(
```

Comment:
Is strtr doing one pass over the content or multiple passes? I'm worried it could mistreat sequences of `\r\r\n`

Reply:
I tested to confirm. `\r\r\n` [is transformed to two CSS unicode escaped newlines](https://3v4l.org/eGfhZ) `string(6) "\A \A "`, the expected result. The first `\r` is transformed and the following `\r\n` are transformed as one.